src/NodaTime.Test/OffsetDateTest.cs

```diff
@@ -148,5 +151,39 @@ public void Deconstruction()
                 Assert.AreEqual(offset, actualOffset);
             });
         }
+
+        [Test]
+        public void BinarySerialization()
+        {
+            var date = new LocalDate(2013, 4, 12);
+            var offset = Offset.FromHoursAndMinutes(5, 30);
+            var value = new OffsetDate(date, offset);
+            TestHelper.AssertBinaryRoundtrip(value);
+        }
+
+        [Test]
+        public void XmlSerialization_Iso()
```

Comment:
Could you add tests covering BCE?

Specifically, for OffsetDate, I assume that the serialized form of January 1st, 2 BCE would be something like `-0001-01-01Z`, following (I assume) ISO 8601:2000, and not e.g. the craziness discussed in https://www.w3.org/TR/xmlschema-2/#dateTime.

Reply:
Done, thanks. (And yes, we're following ISO-8601.)